app/views/users/partials/ip.blade.php

```diff
@@ -0,0 +1,3 @@
+@if(Auth::user()->isAdmin())
```

Comment:
To stop the 500 error, when opening the Profil page without Login use this Code:

```
@if (Auth::check())
	@if(Auth::user()->isAdmin())
		IP: {{{ $user->ip }}}
	@endif
@endif
```